tests/cephfs/clients/validate_root_squash.py

```diff
@@ -0,0 +1,493 @@
+    5. Mount using kernel and fuse with above clients on 2 Virtual Machines
+       1. Mount 1: Kernel client_1
+       2. Mount 2: Fuse Client_1
+       3. Mount 3: kernel Client_2
+       4. Mount 4: Fuse Client_2
+       5. Mount 5: kernel Client_3
+       6. Mount 6: Fuse Client_3
+       7. Mount 7: kernel Client_4
+       8. Mount 8: Fuse Client_4
+    6. Perform below operations on Above mounts on both Virtual Machines
+    """
+    try:
+        tc = "CEPH-83591419"
+        log.info("Running cephfs %s test case" % (tc))
+        fs_util = FsUtils(ceph_cluster)
+        config = kw.get("config")
+        build = config.get("build", config.get("rhbuild"))
+        clients = ceph_cluster.get_ceph_objects("client")
+        client, client1 = clients[0], clients[1]
+        if len(clients) < 2:
+            log.info(f"This requires atleast 2 clients total clients : {len(clients)}")
+            log.error(f"This requires atleast 2 clients total clients : {len(clients)}")
+            return -1
+        log.info("checking Pre-requisites")
+        fs_util.prepare_clients(clients, build)
+        fs_util.auth_list(clients)
+
+        log.info(
+            "Create Test configuration - Additional CephFS and one subvolume in each CephFS"
+        )
+
+        fs_list = ["cephfs_1", "cephfs_2"]
+        subvolume_group_list = ["subvol_group_1", "subvol_group_2"]
+        subvolume_list = [
+            {"subvol_name": "subvol_1"},
+            {"subvol_name": "subvol_2"},
+            {"subvol_name": "subvol_3", "group_name": "subvol_group_1"},
+            {"subvol_name": "subvol_4", "group_name": "subvol_group_2"},
+        ]
+        [fs_util.create_fs(client, fs_name) for fs_name in fs_list]
+        for fs in fs_list:
+            for subvolume_group in subvolume_group_list:
+                subvolumegroup = {"vol_name": fs, "group_name": subvolume_group}
+                fs_util.create_subvolumegroup(client, **subvolumegroup)
+            for subvolume in subvolume_list:
+                fs_util.create_subvolume(client, vol_name=fs, **subvolume)
+        fs_util.create_ceph_client(
+            client,
+            "client_x",
+            mon_caps="allow *",
+            mds_caps="allow *",
+            osd_caps="allow *",
+        )
+        fs_util.create_ceph_client(
+            client,
+            "client_1",
+            mon_caps="allow *",
+            mds_caps="allow * root_squash",
+            osd_caps="allow *",
+        )
+        fs_util.create_ceph_client(
+            client,
+            "client_2",
+            mon_caps="allow *",
+            mds_caps=f"allow * fsname={fs_list[0]} root_squash",
+            osd_caps="allow *",
+        )
+        fs_util.create_ceph_client(
+            client,
+            "client_3",
+            mon_caps="allow *",
+            # mds_caps=f"allow rw fsname={fs_list[0]} root_squash,allow * path=/,allow *
+            # fsname={fs_list[1]} root_squash path=/",
```

Comment:
Are the commented lines going to be retained during merge? If yes, please add the reason, else please remove it.

Reply:
Added the comment suma